opentracing_instrumentation/client_hooks/strict_redis.py

```diff
@@ -63,15 +63,38 @@ def peer_tags(self):
     redis.StrictRedis.peer_tags = peer_tags
 
     old_execute_command = redis.StrictRedis.execute_command
+    old_get = redis.StrictRedis.get
+    old_set = redis.StrictRedis.set
+    old_setex = redis.StrictRedis.setex
+
+    def get(self, key):
+        self._extra_tags = [('redis.key', key)]
```

Comment:
So the way that `StrictRedis` works is kind of magical. When you call `SetRedis.foo(...)` it then dispatches to `StrictRedis.execute_command("foo", ...)` by inspecting its own function name.

Therefore when you call a method like `get()` on the `StrictRedis` class, first it calls `get()`, then that immediately dispatches `execute_command()`, and then that has the bits that actually send data over the wire to redis and then waits for them to get back. That's why I'm doing the weird thing where I store the attributes in `_extra_tags` in the command method, and then use them in `execute_command`.